pom.xml

```diff
@@ -58,6 +58,7 @@
             <ratchetFrom>NONE</ratchetFrom>
             <!-- define a language-specific format -->
             <java>
+              <toggleOffOn />
```

Comment:
@srl295 I'm a little surprised that cldr didn't need this, which enables spotless:on/off comments for disabling formatting of a few long arrays... probably those arrays should be implemented differently anyway but I think this is better than one item per line...

Reply:
CLDR probably needs this but didn't know about it. we just let spotless do what it wants to do. want to file an issue and/or put on the infra agenda?

Reply:
I'll put it on the agenda. I don't remember any specific instances of anyone being unhappy with the spotless results. I just vaguely recall such issues when various formatting options were being considered.

Reply:
anyway is this PR okay?